src/l500/l500-private.h

```diff
@@ -214,16 +214,17 @@ namespace librealsense
 
         enum gvd_fields
         {
-            fw_version_offset = 12,
-            is_camera_locked_offset = 33,
-            module_serial_offset = 56,
-            module_asic_serial_offset = 72
+            fw_version_offset = 12,         // FunctionalPayloadVersion
+            is_camera_locked_offset = 6,    // eyeSafety: encompasses eeprom, flash, & registers
+            module_serial_offset = 60,      // "OpticalHeadModuleSN" -> RS2_CAMERA_INFO_SERIAL_NUMBER
+            module_asic_serial_offset = 74  // RS2_CAMERA_INFO_ASIC_SERIAL_NUMBER & RS2_CAMERA_INFO_FIRMWARE_UPDATE_ID
```

Comment:
The modified offsets is an API break with the previous FW version - unless the functionality didn't work in previous releases.

Reply:
It sort of worked (showed the right values) but only by accident